docs/tools_scripts.md

```diff
@@ -312,3 +151,202 @@ The table below lists the models that are guaranteed to be exportable to TensorR
 - *All models above are tested with Pytorch==1.8.1,  onnxruntime==1.7.0 and tensorrt==7.2.3.4*
 - If you meet any problem with the listed models above, please create an issue and it would be taken care of soon. For models not included in the list, please try to solve them by yourself.
 - Because this feature is experimental and may change fast, please always try with the latest `mmcv` and `mmedit`.
+
+### Evaluate ONNX and TensorRT Models (experimental)
+
+We provide methods to evaluate TensorRT and ONNX models in `tools/deploy_test.py`. To evaluate ONNX models, onnxruntime should be installed first.
+
+#### Prerequisite
+
+- Install onnx and onnxruntime-gpu
+
+  ```shell
+  pip install onnx onnxruntime-gpu
+  ```
+
+#### Usage
+
+```bash
+python tools/deploy_test.py \
+    ${CONFIG_FILE} \
+    ${MODEL_PATH} \
+    ${BACKEND} \
+    --out ${OUTPUT_FILE} \
+    --save-path ${SAVE_PATH} \
+    ----cfg-options ${CFG_OPTIONS} \
+```
+
+#### Description of all arguments
+
+- `config`: The path of a model config file.
+- `model`: The path of a TensorRT or an ONNX model file.
+- `backend`: The backend for testing, choose tensorrt or onnxruntime.
+- `--out`: The path of output result file in pickle format.
+- `--save-path`: The path to store images and if not given, it will not save image.
+- `--cfg-options`: Override some settings in the used config file, the key-value pair in `xxx=yyy` format will be merged into config file.
+
+#### Results and Models
+
+<table class="tg">
```

Comment:
Please keep the format of html table the same as before. For instance, the `class` attribute.
You can check generated HTML pages by following instructions:
- Generate HTML pages.
```bash
pip install recommonmark sphinx_markdown_tables sphinx_rtd_theme
cd docs
make html
```
- Then view the related HTML files with the browser.

Reply:
Well, I used [Table Generator](https://www.tablesgenerator.com/html_tables). But your advise will be taken.